### PR TITLE
Added mobile parameter to init_transaction method

### PR DESCRIPTION
**Pull request by sassanh**
Added mobile parameter to init_transaction method and changed error messages for more readability.

### CODE REVIEW

File: pay_ir/api/client.py [hunk 1]
```diff
@@ -24,15 +24,15 @@ def __init__(self, api_key):
...
+    def init_transaction(self, amount, redirect_url, factor_number=None, mobile=None, description=None):
         """
             Sends an initial request to 'API_URL_SEND' to retrieve a
                 transaction id. Sent data contains these variables:
 
                     - api (string): You API key for pay.ir
                     - amount (integer): Transaction amount in rials, should be greater than 1000.
                     - redirect (string): Urlencoded return address registered in pay.ir panel.
-                    - factorNumber (string): (optional).
+                    - factor_number (string): (optional).
```
Comment (moeenz) [inline pay_ir/api/client.py]: Please add `mobile` and `description` to function doctoring.

File: pay_ir/api/client.py [hunk 2]
```diff
@@ -24,15 +24,15 @@ def __init__(self, api_key):
...
```
Comment (moeenz) [inline pay_ir/api/client.py:35]: Please add mobile and description to function docstring.

Comment (sassanh) [reply pay_ir/api/client.py:35]: Done

Comment (sassanh) [reply pay_ir/api/client.py:35]: @moeenz is there anything else I can do about this PR?

Comment (moeenz) [reply pay_ir/api/client.py:35]: Merged.
Sorry for the delay.

Comment (sassanh) [reply pay_ir/api/client.py:35]: Thanks! It's alright.

**Pull request merged**